### ISSUE TITLE
Winkey stopped working

### ISSUE THREAD

**Issue by ok1cdj** [magicbug/Cloudlog#2529]
There is not possible to select port when you clicked on Connect button, also there is problem to save macros. Dialog with Macros Saved appears then click Save and nothing. Can you check please...

**Comment by AndreasK79** [comment]
Yeah, seems some problems there. #2540 is a work in progress to fix.

**Comment by AndreasK79** [comment]
@ok1cdj could you test latest dev to see if that fixes the problem?

**Comment by magicbug** [comment]
Should help but I think it will still need some work to reconnect to the comport on page reload

**Comment by GregTroar** [comment]
The issue I have is that the macros are not being saved at all.
You put a name and the text:

CQ

CQ de F4BPO F4BPO K

Click Save then close
Re open the window everything is blank

It is not saving macros at all
